### PR TITLE
Catalan Localisation

### PR DESCRIPTION
**Pull request by VentiPlenti**
## What are the changes?

Modified the Catalan localisation to include translations for the Natures, Types, Statuses, Menus, Biomes, and Common terms. These are mostly displayed on the main menu and summary screens.

## Why am I doing these changes?

These are key components of the Catalan localisation. The Catalan localisation was a copy of English previously

## What did change?

Completed translations for:

common.ts
nature.ts
menu.ts
pokemon-info.ts
arena-flyout.ts
biomes.ts

### Screenshots/Videos

image

image

image

## How to test the changes?

Change the language in the game to "Català" in the settings. 

## Checklist
- [x] **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I considered writing automated tests for the issue?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/ca_ES/menu.ts [hunk 1]
```diff
@@ -6,57 +6,57 @@ import { SimpleTranslationEntries } from "#app/interfaces/locales";
...
+  "registrationAgeWarning": "En registrar-te, confirmes que tens 13 anys o més.",
+  "backToLogin": "Torna a Iniciar Sessió",
+  "failedToLoadSaveData": "No s'han pogut carregar les dades desades. Torneu a carregar la pàgina.\nSi això continua, comproveu #announcements a Discord.",
+  "sessionSuccess": "Sessió carregada amb èxit.",
+  "failedToLoadSession": "No s'han pogut carregar les dades de la sessió.\nÉs possible que estiguin malmeses.",
+  "boyOrGirl": "Ets Nen o Nena?",
+  "evolving": "Que?\n{{pokemonName}} està evolucionant!",
+  "stoppedEvolving": "Prou?\nL'evolució de {{pokemonName}} s'ha aturat!",
+  "pauseEvolutionsQuestion": "Vols aturar les evolucions de {{pokémon Name}}?\nSempre poden ser activades des de la pantalla del teu equip.",
   "evolutionsPaused": "Evolutions have been paused for {{pokemonName}}.",
```
Comment (CodeTappert) [inline src/locales/ca_ES/menu.ts:41]: Why didnt you translate this one?

Comment (VentiPlenti) [reply src/locales/ca_ES/menu.ts:41]: Whoops that was an oversite on my end, I'll get that done now

**Review by CodeTappert — APPROVED** [review]

**Comment by Tempo-anon** [comment]
Please fix merge conflicts

**Comment by Adri1** [comment]
Conclifts resolved in #4194

**Pull request closed without merging**